test/e2e/ps_test.go

```diff
@@ -181,6 +181,18 @@ var _ = Describe("Podman ps", func() {
 		Expect(result.OutputToStringArray()[0]).To(Equal(fullCid))
 	})
 
+	It("podman ps multiple filters", func() {
+		session := podmanTest.Podman([]string{"run", "-d", "--name", "test1", "--label", "key1=value1", ALPINE, "top"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+		fullCid := session.OutputToString()
+
+		result := podmanTest.podman([]string{"ps", "-aq", "--no-trunc", "--filter", "name=test1", "--filter", "label=key1=value1"})
+		result.WaitWithDefaultTimeout()
+		Expect(result.ExitCode()).To(Equal(0))
+		Expect(result.OutputToStringArray()[0]).To(Equal(fullCid))
+	})
+
```

Comment:
I think this new test might have succeeded with the old behavior.  It might be good to create a second container with key1==value2 and then make sure you only get one container back.